routes/web.php

```diff
@@ -29,6 +29,7 @@
 <<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```
phần này rebase lỗi à e, check lại giúp a nhé

Reply:
Vâng ạ.